junit-platform-commons/src/main/java/org/junit/platform/commons/util/AnnotationUtils.java

```diff
@@ -258,6 +258,10 @@ else if (candidate.annotationType().equals(containerType)) {
 		// @formatter:on
 	}
 
+	public static List<Method> findAnnotatedMethods(Class<?> clazz, Class<? extends Annotation> annotationType) {
+		return findAnnotatedMethods(clazz, annotationType, MethodSortOrder.HierarchyDown);
+	}
+
```

Comment:
I introduced it because there's a similar convenient overload at `ReflectionUtils:611`:
```
public static List<Method> findMethods(Class<?> clazz, Predicate<Method> predicate) {
	return findMethods(clazz, predicate, MethodSortOrder.HierarchyDown);
}
```